### PR TITLE
Feature/room config and anteroom conection

### PR DESCRIPTION
**Pull request by NicolasLLans**
# Description:

configuration to load the room creator player data. dynamic avatar, dynamic name, dynamic maxPlayers.

# Trello link:

[https://trello.com/c/7pliEbQS/<Edit This>](https://trello.com/c/vKhVvTms/83-integraci%C3%B3n)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Captura de pantalla 2024-04-25 142836

Captura de pantalla 2024-04-25 142917

Captura de pantalla 2024-04-25 143013

### CODE REVIEW

File: src/app/pages/anteroom/anteroom.component.html [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
           <div class="players d-flex flex-column justify-content-center align-items-center">
             <img class="avatar" [src]="item.avatar" [alt]="item.name">
             {{item.name}}
-            <div>{{item.estado}}</div>
+            <div class="onBar offBar"></div>
           </div>
         }
       </div>
-      <app-custom-button>Preparado</app-custom-button>
+      <app-custom-button (click)="changeEstado()">Preparado</app-custom-button>
```
Comment (WanderleeDev) [inline src/app/pages/anteroom/anteroom.component.html:20]: nombre del método se maneje solo en ingles sin mezclar español. Se recomienda  cambiar a (changeStatus)

File: src/app/shared/services/web.socket.api.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        const response = _this.stompClient.send(
+          `/app/rooms.join/${roomId}`,
+          {},
+          JSON.stringify({ player_name: player_name, avatar_id: avatar_id })
+        );
+        console.log('response stomp send: ', response);
+      },
+      this.errorCallBack
+    );
+    // if (this.stompClient.status === 'CONNECTED') {
```
Comment (WanderleeDev) [inline src/app/shared/services/web.socket.api.ts:46]: Consulta  ¿Se requerirá a futuro este bloque de código?

**Review by Alejandro25AR — CHANGES_REQUESTED** [review]

File: src/app/shared/services/web.socket.api.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    //     let response = this.stompClient.send(`/app/rooms.join/${roomId}`, {}, JSON.stringify({'player_name': player_name, 'avatar_id': avatar_id}));
+    //     console.log("response stomp send: ", response);
+    // }
+  }
```
Comment (WanderleeDev) [inline src/app/shared/services/web.socket.api.ts:50]: This block of code will be used in the future

Comment (NicolasLLans) [reply src/app/shared/services/web.socket.api.ts:50]: maybe

**Comment by NicolasLLans** [comment]
change requested ready

**Review by WanderleeDev — APPROVED** [review]

**Review by Alejandro25AR — APPROVED** [review]

**Pull request merged**